code/modules/error_handler/error_viewer.dm

```diff
@@ -112,7 +112,6 @@ var/global/datum/ErrorViewer/ErrorCache/error_cache = null
 
 	// Show the error to admins with debug messages turned on, but only if one
 	//  from the same source hasn't been shown too recently
-	// (Also, make sure config is initialized, or log_debug will runtime)
```

Comment:
Okay, so I replaced my entire error_viewer.dm manually by copying the raw from the git, since I couldn't find the button to download it. However, now it simply says there's no changes on my git and so I can't commit it.